app/views/courses/show.html.haml

```diff
@@ -55,7 +55,7 @@
           = link_to @course.external_syllabus, @course.external_syllabus, :target => "blank"
     %li
       %span.summary-label Goal for Class Visit:
-      = @course.goal.html_safe unless @course.goal.nil?
+      = sanitize @course.goal.html_safe unless @course.goal.nil?
```

Comment:
I believe html_safe should be removed from this - sanitize is meant to be used _instead_ of html_safe, not in addition. if you'll look at the example code here: https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html,